### PR TITLE
Updated Log Analytics recommendations based on verification from PG

### PR DESCRIPTION
**Pull request by judyer28**
# Overview/Summary

Updated Log Analytics recommendations based on discussions with the PG.  Original LOG-2 and LOG-3 have been removed. 

## Related Issues/Work Items

[AB#34111](https://dev.azure.com/CSUSolEng/Process%20Guidance/_workitems/edit/34111)

## This PR fixes/adds/changes/removes

1. Update category of LOG-1 to Governance
2. Remove original LOG-2 and LOG-3
4. Update LOG-5 to category to Governance

### Breaking Changes

NA

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by ejhenry — COMMENTED** [review]
Thanks @judyer28! We're not changing recommendation IDs, so no need to re-number 4 & 5. We found changing IDs was resulting in a lot of confusion.

**Comment by judyer28** [comment]
Thanks @ejhenry.  Reverted back to original ID numbers.

File: docs/content/services/monitoring/log-analytics/_index.md [hunk 1]
```diff
@@ -33,13 +31,13 @@ Definitions of states can be found [here]({{< ref "../../../_index.md#definition
...
 
-**Category: Disaster Recovery**
+**Category: Governance**
 
 **Impact: Medium**
 
 **Guidance**
 
-Data export in a Log Analytics workspace lets you continuously export data to an Azure Storage account.  Protect your Log Analytics workspace data from the unlikely event of a regional failure by continuously exporting to a geo-redundant storage (GRS) or geo-zone-redundant storage (GZRS) account.
+Data export in a Log Analytics workspace lets you continuously export data to an Azure Storage account.  Protect your Log Analytics workspace data from the unlikely event of a regional failure by continuously exporting to a geo-redundant storage (GRS) or geo-zone-redundant storage (GZRS) account.  This is primarily a recommendation to meet compliance for data retention.
```
Comment (oZakari) [inline docs/content/services/monitoring/log-analytics/_index.md:40]:
```suggestion
Data export in a Log Analytics workspace lets you continuously export data to an Azure Storage account.  Protect your Log Analytics workspace data from the unlikely event of a regional failure by continuously exporting to a geo-redundant storage (GRS) or geo-zone-redundant storage (GZRS) account.  This is primarily a recommendation to meet compliance for data retention, but can also be used to integrate the data with other Azure services and tools. 
```

**Review by oZakari — CHANGES_REQUESTED** [review]
Just a minor addition to the wording for the recommendation if it makes sense to you?

**Comment by judyer28** [comment]
Makes sense to me @oZakari.  Thanks for the review and suggestion.

**Review by oZakari — APPROVED** [review]

**Review by ejhenry — APPROVED** [review]
LGTM 👍

**Pull request merged**